.yamllint.yml

```diff
@@ -12,15 +12,15 @@ extends: default
 rules:
   braces:
     forbid: non-empty
-    min-spaces-inside: 1
-    max-spaces-inside: 1
+    min-spaces-inside: 0
```

Comment:
Why ?

Reply:
My IDE is configured to auto format on save.

For yaml file it remove the space inside the bracket / braces, I didn't find a way to configure it.

Reply:
@mmmarcos any though on this ?

Reply:
Avoiding spaces immediately before bracket/braces feels natural to me because of Python. Also, it seems to be case for yml examples I found online. So :man_shrugging: